maintainer.schema.json

```
@@ -0,0 +1,131 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://forklore.org/maintainer.schema.json",
+  "title": "Maintainer",
+  "description": "Schema for validating maintainer JSON files",
+  "type": "object",
+  "required": ["username", "full_name", "photo", "socials", "projects", "form"],
+  "additionalProperties": false,
+  "properties": {
+    "username": {
+      "type": "string",
+      "description": "Unique identifier for the maintainer",
+      "minLength": 1
+    },
+    "full_name": {
+      "type": "string",
+      "description": "Full name of the maintainer",
+      "minLength": 1
+    },
+    "photo": {
+      "type": "string",
+      "description": "URL to the maintainer's photo",
+      "format": "uri"
+    },
+    "designation": {
+      "type": "string",
+      "description": "Job title or role description"
+    },
+    "socials": {
+      "type": "array",
+      "description": "List of social media links",
+      "items": {
+        "type": "object",
+        "required": ["label", "link"],
+        "additionalProperties": false,
+        "properties": {
+          "label": {
+            "type": "string",
+            "description": "Social media platform name",
+            "enum": [
+              "GitHub",
+              "GitLab",
+              "Gitlab",
+              "Codeberg",
+              "BitBucket",
+              "LinkedIn",
+              "X",
+              "Twitter",
+              "Mastodon",
+              "Bluesky",
+              "BlueSky",
```

```suggestion
              "Bluesky",
```

The canonical name is "Bluesky" in sentence case